tests/test_validation.py

```diff
@@ -0,0 +1,72 @@
+import eido
+import pytest
+from pephub.dependencies import *
+pytest_plugins = ('pytest_asyncio',)
+
+# test project file on schema file validation success
+def test_file_file_validate_valid(project_object_file, schema_file_path):
```

Comment:
It would be nice to add everything to TestValidation class. But overall it's amazing that we have tests:D Thank